### DISCUSSION TITLE
URLs - changing and creating

### DISCUSSION THREAD

**Discussion by sregier206** [GitbookIO/community#828] [Feature Requests]
I need stable URLs for my docs. I have to revamp the setup of our entire knowledge base, which means everything will move and all the URLs will change. WHY?

Why do they need to include the location including page, group, title, etc? All of these can change, and then all the indexing and searching is blown. Can it now be a shorter, more generic URL that does not change? For example, an MS URL is (https://support.microsoft.com/en-us/office/add-and-edit-text-ed1e3147-a846-41ca-8087-49e324cb50bd) so it lists the main URL and the title but not the section or subsection).

This would allow me to move docs and not lose the SEO information. It would also be nice to have the option to keep the URL the same even if I tweak the title, as the title change could be minor, but the meaning is the same.

Thanks - Sherri

**Comment by JaniceManwiller** [comment]
+1 on not including the group and parent pages in the URL for an individual page.

I usually use the **Rename** (for groups/pages) and **Edit Anchor** (for sections) to set the "name" to something that is more like an identifier. So if the displayed title is "Managing widgets", I'll set the "identifier", which displays in the URL, to something like `widgets-manage`. That remains the same through any changes to the displayed title.
